src/twisted/conch/scripts/ckeygen.py

```diff
@@ -96,12 +105,13 @@ def handleError():
     raise
 
 
-
+@_support('rsa')
```

Comment:
I don't love this method name. Might be better to call it `_keyGenerator('rsa')`?